src/main/resources/application.properties

```diff
@@ -1 +1,26 @@
 server.port=8090
+
+# datasource PostgreSQl
```

Comment:
Je hebt je properties netjes gestructureerd staan. Mischien kun je eens kijken om hier yaml te gebruiken, dat is (naar mijn mening) een wat gestructureerder alternatief voor de properties.